src/components/AnnotationViewer.vue

```diff
@@ -292,6 +311,82 @@ export default class AnnotationViewer extends Vue {
       this.drawNewConnections(displayedIds);
     }
     this.annotationLayer.draw();
+    this.drawTooltips();
+  }
+
+  drawTooltips() {
+    const oldFeatures = this.textLayer.features();
+
+    if (this.shouldDrawTooltips) {
+      // Avoid checking the store for each annotation
+      const all = this.tooltipOnAll;
+      const selected = this.tooltipOnSelected;
+      const hovered = this.tooltipOnHovered;
+
+      const displayedAnnotations = this.annotationLayer
+        .annotations()
+        .map((a: any) => a.options("storedAnnotation"))
```

Comment:
Please use a better variable name: a -> annotation ?

Reply:
I used the same naming as in the function `drawAnnotations` above, should I rename in this function too?

Reply:
not necessarily